bash_103_combining_commands/bash_103_combining_commands.md

```diff
@@ -83,9 +78,9 @@ Please download a fresh copy of these files. If you have downloaded them for a p
 
 Download the [`learning_bash` directory](https://github.com/arcus/learning_bash) from GitHub. Once you go to the link:
 
-1. Click on the green **Code** button. There are two "Code" buttons! The one directly below `arcus/learning_bash` functions more like a "home" button. One line below is a dropdown menu labeled `main` and to the right of `main` are three buttons: `Go to File`, `Add File`, and `Code`. This is the **Code** button you want.
+1. Click on the green **Code** drop-down button towards the right-hand side of the screen. 
 
-2. Select **Download ZIP** from the options that appear on clicking the **Code** button.
+2. Select **Download ZIP**.
```

Comment:
```suggestion
2. Select **Download ZIP** from the options that appear.
```
I like having a bit more of the context, but I don't have a good rationale to back this up, so feel free to do whatever you prefer!

Reply:
Adopted!